net/wireguard-go/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                wireguard-go
+version             0.0.20180613
```

Comment:
The version number is a bit eccentric here. I presume this is how WireGuard numbers things?